widgetssdk/src/main/java/com/glia/widgets/di/UseCaseFactory.java

```diff
@@ -772,4 +772,8 @@ DecodeSampledBitmapFromInputStreamUseCase createDecodeSampledBitmapFromInputStre
     public ResetSurveyUseCase createResetSurveyUseCase() {
         return new ResetSurveyUseCase(surveyStateManager, repositoryFactory.getGliaSurveyRepository());
     }
+
+    public void reset() {
```

Comment:
NIT: would be better to have a more descriptive name, currently it is not clear what `usecaseFactory.reset()` should reset. Maybe `resetSurveyState`?

Reply:
@DavDo Thank you. The idea was to have a function where we can add more code that will reset something later on. That's why I used a generic name, not a specific one. Does it make sense to you?

I can still rename it to `resetSurveyState()` because, at this point, this function resets only Survey, you are right. Do you insist on renaming to `resetSurveyState()`?

Reply:
Not at all, but still I think we need to find some descriptive name(e.g. `resetState`), because from the current name I have an impression that it should reset the `UseCaseFactory`

Reply:
Agree. I also thought about `resetState()`
Renamed: 79d45217c2dc8c54d47889199e63cdc882b55042